### PR TITLE
check if project is removed while uploading

### PR DESCRIPTION
**Pull request by kunkun-tang**
We found that upload API doesn't return an error when a deleted project zip is uploaded. 

When a project is deleted, Azkaban just marks it to be inactive today. In this PR, we fix this issue by adding the check. 410 Gone client error response code indicates that access to the target resource is no longer available at the origin server and that this condition is likely to be permanent.

### CODE REVIEW

**Comment by HappyRay** [comment]
Thanks for fixing this.

Can you think of a reason why the project is not deleted?

**Comment by kunkun-tang** [comment]
Do you mean the original design choice why projects are not deleted. Actually, I cannot think of valid points for that. Projects should be entirely deleted IMO. @HappyRay

**Comment by HappyRay** [comment]
@jamiesjc I remember we had a discussion about this behavior before. This behavior causes some other issues too if I remember correctly.

Do you remember the outcome of that discussion?

**Comment by jamiesjc** [comment]
@HappyRay I remember last time we were talking about the upload project issue caused by inconsistency of project cache and DB table. 
I agree the project should be deleted instead of setting it to inactive. I couldn't think of the case where inactive projects will be needed.

**Comment by HappyRay** [comment]
@jamiesjc Thanks.

Shall we try removing the inactive project logic then? This will simplify the code a bit.

**Comment by HappyRay** [comment]
We can still make this change for now.

File: azkaban-web-server/src/main/java/azkaban/webapp/servlet/ProjectManagerServlet.java [hunk 1]
```diff
@@ -1671,6 +1671,11 @@ private void ajaxHandleUpload(final HttpServletRequest req, final HttpServletRes
     final User user = session.getUser();
     final String projectName = (String) multipart.get("project");
     final Project project = this.projectManager.getProject(projectName);
+    if (project.isActive() == false) {
```
Comment (jamiesjc) [inline azkaban-web-server/src/main/java/azkaban/webapp/servlet/ProjectManagerServlet.java]: In CRT deployment, `uploadProject` will be called inside `azkaban.restli.ProjectManagerResource#deploy`. Do we also need to add the same fix there?

Comment (kunkun-tang) [reply azkaban-web-server/src/main/java/azkaban/webapp/servlet/ProjectManagerServlet.java]: Good found! Let me fix it.

**Pull request merged**